frontend/src/features/map_planning/layers/BaseLayer.tsx

```diff
@@ -0,0 +1,97 @@
+import { MAP_PIXELS_PER_METER } from '../utils/Constants';
+import { useQuery, UseQueryResult } from '@tanstack/react-query';
+import { useRef } from 'react';
+import { Layer, Image } from 'react-konva';
+import { FileStat, ResponseDataDetailed, WebDAVClient } from 'webdav';
+
+interface BaseLayerProps {
```

Comment:
You can extend the props from Konva.LayerConfig.
So the default props like 'opacity' and 'visible' get passed automatically.
You can use the PlantsLayer.tsx as example

Reply:
This is still open?

Reply:
Should now be fixed.